frontend/app/services/subscription-service.server.ts

```diff
@@ -23,15 +23,17 @@ type SubscriptionInfo = z.infer<typeof subscriptionInfoSchema>;
 export const getSubscriptionService = moize(createSubscriptionService, { onCacheAdd: () => log.info('Creating new subscription service') });
 
 function createSubscriptionService() {
+  const { CDCP_API_BASE_URI } = getEnv();
+
   async function getSubscription(sin: string) {
     const auditService = getAuditService();
     const instrumentationService = getInstrumentationService();
     auditService.audit('alert-subscription.get', { sin });
-
+    //76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6
     // TODO: "IT-Security won't like SIN being passed as identifier"
-    // TODO: add CDCP_API_BASE_URI
+
     const userId = sin;
```

Comment:
This function should accept the `userId` as a parameter rather than `sin` rather than reassigning a `userId` local variable. The `userId` can also be used for the `audit(..)

The `TODO` comment involving the SIN can be removed (as well as the random commented out UUID).

Reply:
changed to userId